lib/src/cmaf/ham/mapper/hls/mapHlsToHam.ts

```diff
@@ -149,17 +191,12 @@ function mapHlsToHam(manifest: Manifest) {
 		} as SwitchingSet);
 	});
 
-	selectionSets.push({
-		id: uuid(),
-		switchingSets: switchingSetVideos,
-	} as SelectionSet);
-
-	return [{ id: uuid(), selectionSets: selectionSets }];
+	return switchingSetVideos;
 }
 
 function _formatSegments(segments: any[]) {
 	const formattedSegments: Segment[] = [];
-	segments.map(async (segment: any) => {
+	segments.map((segment: any) => {
```

Comment:
Why are we deleting async?

Reply:
Async should be used only in the case where the code inside will `await` for promises.